RecoVertex/BeamSpotProducer/plugins/BeamSpotToCUDA.cc

```diff
@@ -59,19 +74,21 @@ class BeamSpotToCUDA : public edm::global::EDProducer<edm::StreamCache<BeamSpotH
 
 private:
   const edm::EDGetTokenT<reco::BeamSpot> bsGetToken_;
-  const edm::EDPutTokenT<cms::cuda::Product<BeamSpotCUDA>> bsPutToken_;
+  const edm::EDPutTokenT<cms::cuda::Product<BeamSpotCUDA> > bsPutToken_;
```

Comment:
```suggestion
  const edm::EDPutTokenT<cms::cuda::Product<BeamSpotCUDA>> bsPutToken_;
```

Reply:
Thanks, I wonder how that happened. Ok, apparently in `slc7_amd64_gcc10` `clang-format` prefers `> >` over `>>` for some reason.

Reply:
Ouch... having `clang-format` suggest different changes on different architectures/compilers is a worrying thought... as much as these are just whitespaces, it could lead to a lot of back-and-forth changes.